app/connectors/EventReportingConnector.scala

```diff
@@ -229,6 +235,24 @@ class EventReportingConnector @Inject()(
         }
       }
   }
+
+  def getListOfVersions(pstr: String, startDate: String)(implicit headerCarrier: HeaderCarrier, ec: ExecutionContext): Future[Seq[VersionsWithSubmitter]] = {
+    val logger = Logger(classOf[EventReportingConnector])
```

Comment:
logger should be outside of this function as private val logger.

Reply:
Also executionContext implicit on function should be replaced by injecting implicit into class.